paddle/operators/crop_op.h

```diff
@@ -0,0 +1,138 @@
+/* Copyright (c) 2016 CropdleCropdle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#pragma once
+
+#include "paddle/framework/eigen.h"
+#include "paddle/framework/op_registry.h"
+
+namespace paddle {
+namespace operators {
+
+template <typename T, size_t D, int MajorType = Eigen::RowMajor,
+          typename IndexType = Eigen::DenseIndex>
+using EigenTensor = framework::EigenTensor<T, D, MajorType, IndexType>;
+
+using Tensor = framework::Tensor;
```

Comment:
follow the name convention of the google style [click here in detail](https://google.github.io/styleguide/cppguide.html#Namespaces). we should not use any `using` in the header file.

Reply:
Thx.

Reply:
In addition, Tensor has no template parameter, Just write
```c++
using framework::Tensor;
```
is okay.

Reply:
Fixed.